src/main/webapp/WEB-INF/view/profile-pages.jsp

```diff
@@ -21,7 +21,7 @@
   <!--TODO(asma526):
 
   Add an about me box that can only be edited by the user
-  Add a box that displays all messages sent out by the user 
+  Add a box that displays all messages sent out by the user
```

Comment:
Do you plan to do these are part of your MVP? I'd expect them to be in the MVP

Reply:
Yeah sorry I submitted the pull request before I made all the changes.

Reply:
done.